layouts/DocsPage/Header.tsx

```diff
@@ -23,10 +23,25 @@ interface DocHeaderProps {
 const GITHUB_DOCS = process.env.NEXT_PUBLIC_GITHUB_DOCS;
 
 const handleClickScroll = () => {
```

Comment:
Should we delete the "Improve" button instead of scrolljacking the browser? Reasons to remove it:
- Presumably, a reader would have already scrolled to the bottom of a page themselves if they are qualified to leave a rating/comment on how helpful the page was.
- It adds extra steps to the reader's UX. If a reader wants to improve a page, they need to (a) wait for the scrolling to finish, (b) infer the intention of the scrolljacking, and (c) click on a button that does what they already wanted to do with the first button

Reply:
I don't have a strong opinion either way. The scrolljacking is a little clunky. 

Thoughts @benarent ? I don't mind taking it out

Reply:
I think we should remove the button, but also put the feedback below the TOC, so it's more easily visible.

![image](https://github.com/gravitational/docs/assets/559288/c84862e9-a272-405e-87cb-92a2bd2b02d9)

Reply:
<img width="602" alt="Screenshot 2023-07-10 at 6 03 50 PM" src="https://github.com/gravitational/docs/assets/41009873/9ced64d0-aa7c-411b-8be7-51ebf44fb88e">

Moved @benarent 

Also removed the "improve" button

Reply:
@dumez-k , can you make the header the same CSS style as 'Table of Contents

Reply:
@benarent yep!